parla/array.py

```diff
@@ -131,7 +132,6 @@ def clone_here(source, kind=None):
     else:
         raise TypeError("Array required, given value of type {}".format(type(source)))
 
-
```

Comment:
This isn't important since we are in a hurry, but please clean your diffs to remove random changes like this. In this case it actually makes the code match the style guides less well.